src/main/java/TaskGenerator.java

```diff
@@ -0,0 +1,10 @@
+public class TaskGenerator {
+    public static Task generate(String[] parsedInput) {
+        if (parsedInput.length == 1) {
+            return new ToDo(parsedInput[0]);
+        } else if (parsedInput.length == 2) {
+            return new Deadline(parsedInput[0], parsedInput[1]);
+        }
+        return new Event(parsedInput[0], parsedInput[1], parsedInput[2]);
+    }
```

Comment:
Perhaps you can reuse enums that you have defined above to determine the type of task that you are generating?